apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -17,6 +14,31 @@ The information panel is a pattern used primarily in tables. It allows viewing o
 
 The info panel is reserved for complex tables, because it allows the user to gradually explore the content without overwhelming them with too many details. It also helps avoiding the unpleasant horizontal scrolling when the table gets too wide. Not every table requires an information panel.
 
+## Usage
+
+### Composition API
+
+The Information Panel component is made up of customizable subcomponents. Here’s what the basic usage looks like:
+
+```tsx
```

Comment:
nit: new lines in some places might help make the code easier to read. E.g. new line between `<Table />` and `<InformationPanel>`.

Reply:
Maybe a new line between `</InformationPanelBody>` and `</InformationPanel>` too?